pyomo/contrib/mindtpy/single_tree.py

```diff
@@ -657,256 +651,143 @@ def handle_lazy_subproblem_other_termination(
                     )
-                    self.add_lazy_no_good_cuts(var_values, solve_data, config, opt)
-                return
-            elif config.strategy == 'OA':
+                    self.add_lazy_no_good_cuts(var_values, mindtpy_object, config, opt)
                 return
+            # TODO: check why this need to be commented. This previously worked well.
+            # elif config.strategy == 'OA':
+            #     return
         else:
-            solve_data.integer_list.append(solve_data.curr_int_sol)
+            mindtpy_object.integer_list.append(mindtpy_object.curr_int_sol)
 
         # solve subproblem
         # The constraint linearization happens in the handlers
-        fixed_nlp, fixed_nlp_result = solve_subproblem(solve_data, config)
-
+        fixed_nlp, fixed_nlp_result = mindtpy_object.solve_subproblem(config)
         # add oa cuts
         if fixed_nlp_result.solver.termination_condition in {
             tc.optimal,
             tc.locallyOptimal,
             tc.feasible,
         }:
-            self.handle_lazy_subproblem_optimal(fixed_nlp, solve_data, config, opt)
+            self.handle_lazy_subproblem_optimal(fixed_nlp, mindtpy_object, config, opt)
             if (
-                abs(solve_data.primal_bound - solve_data.dual_bound)
+                abs(mindtpy_object.primal_bound - mindtpy_object.dual_bound)
                 <= config.absolute_bound_tolerance
             ):
                 config.logger.info(
                     'MindtPy exiting on bound convergence. '
                     '|Primal Bound: {} - Dual Bound: {}| <= (absolute tolerance {})  \n'.format(
-                        solve_data.primal_bound,
-                        solve_data.dual_bound,
+                        mindtpy_object.primal_bound,
+                        mindtpy_object.dual_bound,
                         config.absolute_bound_tolerance,
                     )
                 )
-                solve_data.results.solver.termination_condition = tc.optimal
+                mindtpy_object.results.solver.termination_condition = tc.optimal
                 return
         elif fixed_nlp_result.solver.termination_condition in {
             tc.infeasible,
             tc.noSolution,
         }:
-            self.handle_lazy_subproblem_infeasible(fixed_nlp, solve_data, config, opt)
+            self.handle_lazy_subproblem_infeasible(
+                fixed_nlp, mindtpy_object, config, opt
+            )
         else:
             self.handle_lazy_subproblem_other_termination(
                 fixed_nlp,
                 fixed_nlp_result.solver.termination_condition,
-                solve_data,
+                mindtpy_object,
                 config,
             )
 
 
 # Gurobi
 
 
-def LazyOACallback_gurobi(cb_m, cb_opt, cb_where, solve_data, config):
+def LazyOACallback_gurobi(cb_m, cb_opt, cb_where, mindtpy_object, config):
     """This is a GUROBI callback function defined for LP/NLP based B&B algorithm.
```

Comment:
Let's also use the chance to unify notation for CPLEX and GUROBI. I like having both upper-case

Reply:
Only in comments? Or inside function names or variable names?

Reply:
I would leave it only in the comments. In the function and variable names, let's follow the convention we are using (Cameback for functions, snake for variables).

Reply:
The previous comment addressed this. Follow the Google convention